### PR TITLE
fix importing tasks

### PR DESCRIPTION
**Pull request by darkyne**
Fix for issue #622
Now only the "create new task" button checks the syntax for the string input.
"add task" button related to already existing files was wrongly checking the string input on top which was not even related to those files.

### CODE REVIEW

File: inginious/frontend/static/js/task_dispensers.js [hunk 1]
```diff
@@ -101,9 +101,9 @@ function dispenser_util_click_modal_task(task) {
     input.attr("checked", !input.attr("checked"));
 }
 
-function dispenser_util_add_tasks_to_section(button) {
+function dispenser_util_add_tasks_to_section(button, checker) {
     task_id= $("#new_task_id").val();
-    if(!task_id.match(/^[a-zA-Z0-9_\-]+$/)){
+    if(checker && !task_id.match(/^[a-zA-Z0-9_\-]+$/)){
```
Comment (Drumor) [inline inginious/frontend/static/js/task_dispensers.js:106]: Why not simply use the value of `existing_task` variable to define if you need to do this verification?

**Review by Drumor — CHANGES_REQUESTED** [review]
I think this can be simplify with existing variable.

File: inginious/frontend/static/js/task_dispensers.js (hunk 1 above)
Comment (darkyne) [reply inginious/frontend/static/js/task_dispensers.js:106]: Damn you are probably right!
I will do this instead :)

**Comment by Drumor** [comment]
In fact there is a tiny indentation issue in the if

**Comment by darkyne** [comment]
No more :)

**Pull request merged**